tests/commonUtils.sh

```diff
@@ -65,6 +65,7 @@ function generate_istio_yaml() {
     mkdir -p ${dest_dir}
     cp ${src_dir}/* ${dest_dir}
     sed -i "s|image: {PILOT_HUB}/\(.*\):{PILOT_TAG}|image: ${PILOT_HUB}/\1:${PILOT_TAG}|" ${dest_dir}/istio-pilot.yaml
+    sed -i "s|args: [\"discovery\", \"-v\", \"2\"]|args: ["discovery", "-v", "2", \"-a\", $NAMESPACE]|" istio-pilot.yaml
```

Comment:
will this be set/configurable in the original istio-pilot.yaml after your bigger PR ? (or maybe for 0.3 we will generate all those yamls from internal templates; all those sed are very unwieldy and customers are also likely to need changes/tweaks)

Reply:
The canonical installation will be cluster wide.

I will add a knob to allow people to generate a namespace only installation.